internal/feedback-tool/src/main/java/org/wso2/security/tool/adapter/ExcelInputAdapter.java

```diff
@@ -0,0 +1,105 @@
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.security.tool.exception.FeedbackToolException;
+import org.wso2.security.tool.util.Constants;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+
+/**
+ * ExcelInputAdapter -- This class exclusively consists of the functionality to convert the data in the data file
+ * uploaded with the .xlsx extension to the JSON format.The method convert() in this class
+ * returns a JSONObject containing all the data in the uploaded data file.
+ */
+public class ExcelInputAdapter implements InputAdapter {
+
+    private static final Logger log = LoggerFactory.getLogger(ExcelInputAdapter.class);
+    private String logInfo;
+
+    /**
+     * Converts the data in the files with .xlsx extension to the JSON format.
+     * A workbook is created from the the excel file (.xlsx) and while iterating through the sheets in the workbook;
+     * the data is read and  set in to a JSONObject. The JSONObject returned by the method contains an array of
+     * row objects corresponding to each row in the workbook. A row object contains values of each cell in a given row,
+     * with key values starting from letter 'A'.
+     *
+     * @param dataFilePath The path where the data file uploaded is saved.
+     * @return returns the JSON object that contains all the data in the .xlsx file.
+     * @throws FeedbackToolException If the .xlsx file is not found in the given path or due to an error in
+     *                               parsing the data in the data file.
+     */
+    @Override
+    public JSONObject convert(String dataFilePath) throws FeedbackToolException {
+
+        // JSONObject to hold the array of row objects
+        JSONObject dataJSONObject = new JSONObject();
+        try {
+            Workbook workbook = WorkbookFactory.create(new File(dataFilePath));
+            logInfo = "Workbook has " + workbook.getNumberOfSheets() + " sheets";
+            log.info(logInfo);
+
+            Iterator<Sheet> sheetIterator = workbook.sheetIterator();
+
+            // JSONArray to hold all the row objects
+            JSONArray rowsJSONArray = new JSONArray();
+            while (sheetIterator.hasNext()) {
+                Sheet sheet = sheetIterator.next();
+                logInfo = "Sheet: " + sheet.getSheetName() + " has " + sheet.getNumMergedRegions()
+                        + " merged regions";
+                log.info(logInfo);
+
+                DataFormatter dataFormatter = new DataFormatter();
+
+                logInfo = "Iterating over Rows and Columns using for-each loop";
```

Comment:
Why do we need to have a variable for log message? Shouldn't we simply place the message with log.info()